src/models/sensor-gdx-manager.ts

```diff
@@ -123,10 +118,38 @@ export class SensorGDXManager extends SensorManager {
       return this.hasData;
     }
 
+    // This value gets sent to gdxDevice.start() to set the measurement period.
+    // It gets used when we've actually started the sensor data graphing.
+    getMeasurementPeriod() {
+      // The Heart Rate sensor is only able to collect 1 sample per second.
+      // Currently HEARTBEAT_INTERVAL_MS is set to 1000ms (1 second).
+      // If that changes we'll need to manually return 1000 here.
+      if (this.gdxDevice.orderCode === "GDX-EKG") {
+        return HEARTBEAT_INTERVAL_MS;
```

Comment:
I think you should go ahead and just return a new constant of `MIN_EKG_INTERVAL_MS` (set to 1000) here (with a comment of why it is 1000 where you create the constant) so if someone does change `HEARTBEAT_INTERVAL_MS` in the future without seeing this comment it doesn't break the sensor.